### ISSUE TITLE
Animate "You Called for Help" display text

### ISSUE THREAD

**Issue by WesLSmith** [TheAfroOfDoom/omega-flowey-minecraft-remastered#137]
Text should unfurl from left to right in a fashion similar to this youtube video https://www.youtube.com/watch?v=ptRk7h_ki_E

This should be accomplishable via creating a large number of block displays. The color of each character can be set to black then through creating and killing thos blocks characters can be revaled on at a time.

**Comment by WesLSmith** [comment]
going to try doing this by hand in mfcuntions using /data merge, idk how committed we are to a flashy solution

**Comment by TheAfroOfDoom** [comment]
@WesLSmith I don't think we can do this with AJ, the ideal solution would allow for any arbitrary text input and unfurl character-by-character

`/data merge` would be a good start for this.
